.github/CODEOWNERS

```diff
@@ -32,16 +31,13 @@ Cargo.toml
 # Engine (old)
 # This section should be removed once the engine moves to /app/engine
 /build.sbt @4e6 @jaroslavtulach @hubertp
-/distribution/ @4e6
+/distribution/ @4e6 @jdunkerley @radeusgd
 /engine/ @4e6 @jaroslavtulach @hubertp
 /project/ @4e6 @jaroslavtulach @hubertp
 /test/ @jdunkerley @radeusgd
-/tools/ @4e6
+/tools/ @4e6 @jaroslavtulach @radeusgd
 
 # Enso Libraries
 # This section should be amended once the engine moves to /app/engine
 /distribution/lib/ @jdunkerley @radeusgd
 /std-bits/ @jdunkerley @radeusgd
-
-# Engine
-/app/engine/ @4e6
```

Comment:
This directory currently does not exist, it shall be re-added once the repository is actually restructured.

Reply:
Please create issue on pivotal for that.

Reply:
> Please create issue on pivotal for that.

OK: https://www.pivotaltracker.com/story/show/183613892